odoo/tools/misc.py

```diff
@@ -47,6 +47,19 @@
 from .float_utils import float_round
 from .which import which
 
+# xlwt and xlsxwriter are monkeypatched if they are installed
+# if they're not installed, don't break on import
```

Comment:
they are not patched if we directly import the tools

Reply:
Actually, these libraries are quite old and could be replaced by openpyxl. Then we could stop monkeypatching them all together. I think it is fine for excel libraries to be patched in init.